torch/distributed/_composable/fsdp/_fsdp_param.py

```diff
@@ -386,12 +387,11 @@ def init_unsharded_param(self):
         if self._unsharded_param is not None:
             assert torch._dynamo.compiled_autograd.compiled_autograd_enabled
             with torch.no_grad():
+                # torch.ops.create_parameter_op.set_(self._unsharded_param, unsharded_param)
                 alloc_storage(self._unsharded_param)
                 self._unsharded_param.copy_(unsharded_param)
```

Comment:
I wanted to change this to `torch.ops.create_parameter_op.set_()` op, but now it also throws segfault error because this custom set_ op also shows up in Compiled Autograd graph and we tried to functionalize it, which introduces the `.copy_` op.